.github/workflows/nightly-builds.yml

```diff
@@ -167,7 +167,14 @@ jobs:
         with:
           report_paths: '**/target/test-reports/TEST-*.xml'
           fail_if_no_tests: false
-          skip_publishing: true
+
+      # Archive test results so we can do some diagnostics later
+      - name: Upload test results
+        uses: actions/upload-artifact@v2
```

Comment:
does this mean that it becomes part of the log archive that can be downloaded?

Reply:
Not sure it's part of the log archive but there will be another place to download.